README.md

```diff
@@ -1,6 +1,28 @@
 # Connect Four
 
-This is the game of Connect Four, playable on the command line. It's currently a work in progress.
+Connect Four is a two-player game similar to tic-tac-toe where the object is to get either four Xs or four Os in a horizontal, vertical, or diagonal line. 
+
+
+<b>Installation</b>
```

Comment:
Headings can use Markdown syntax, so this would be written as `## Installation`.